types/v1/types.go

```diff
@@ -310,8 +243,8 @@ type ObjectReference struct {
 	FieldPath string
 }
 
-//VsmSpec holds the config for creating a VSM
-type VsmSpec struct {
+// VolumeAPISpec holds the config for creating a Volume
+type VolumeAPISpec struct {
```

Comment:
Please raise an issue to remove VolumeAPISpec & use VolumeSpec instead.

Reply:
done !!